pkg/internal/checkup/executor/console/console.go

```diff
@@ -45,24 +52,24 @@ const (
 func NewExpecter(serialConsoleClient vmiSerialConsoleClient,
 	vmiNamespace,
 	vmiName string,
-	timeout time.Duration,
```

Comment:
What is the reason for the move of `timeout` to be a parameter of `spawnConsole`?
(instead of being a member of Expecter)

Reply:
The timeout to spawn the console is different between `LoginToCentOS` (10s) and `SafeExpectBatchWithResponse` (30s). Changing that would be a logic change, and this PR is deliberately avoiding that. It only refactors.
Should we want to choose a unifies timeout we could do so - but in a separate PR.